<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnprPark</class>
 <widget class="QWidget" name="AnprPark">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="parkTabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>810</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; border:0px;</string>
   </property>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Heure d'arriver</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Plaque</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Heure départ</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="depart">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>460</y>
     <width>150</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>Marquer un départ</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0,
                                 stop: 0 #f27436, stop: 0.2 #fbae32,
                                 stop: 0.8 #f27436, stop: 1.0 #f57d36);text-align:left; color : white; border: 0px solid #000028;padding-left:5px;}
QPushButton:hover{;background-color:qlineargradient(x1: 1, y1: 0, x2: 0, y2: 0,
                                 stop: 0 white, stop: 0.2 white ,
                                 stop: 0.8 white, stop: 1.0 #f57d36); border: 0px solid #2f4f4f  ;color:black;}
QPushButton:focus{;background-color:qlineargradient(x1: 1, y1: 0, x2: 0, y2: 0,
                                 stop: 0 white, stop: 0.2 white ,
                                 stop: 0.8 white, stop: 1.0 #f57d36); border: 0px solid #2f4f4f  ;color:white;}</string>
   </property>
   <property name="text">
    <string>Départ</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../IMAGES/Icons/car_out.png</normaloff>../../IMAGES/Icons/car_out.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>31</width>
     <height>31</height>
    </size>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="matPark">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>490</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Schoolbook L</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white;color:#c8cdd3; border:1px solid #3c80e5 ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Chercher par matricule</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="dateFindPark">
   <property name="geometry">
    <rect>
     <x>495</x>
     <y>490</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#252022;text-align:center; color : white; border: 0px solid #000028;}
QPushButton:hover{color:black;}
QPushButton:focus{color:white;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../IMAGES/Icons/whiteCalendar.png</normaloff>../../IMAGES/Icons/whiteCalendar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="matFindPark">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Rockwell</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#252022;text-align:center; color : white; border: 0px solid #000028;}
QPushButton:hover{color:black;}
QPushButton:focus{color:white;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../IMAGES/Icons/search_white.png</normaloff>../../IMAGES/Icons/search_white.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="dateIndicator">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>460</y>
     <width>651</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Semibold</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #517c62 ;color:white;text-align:center;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QDateEdit" name="datePark">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>490</y>
     <width>280</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:white; border:1px solid #3c80e5 ;</string>
   </property>
   <property name="minimumDate">
    <date>
     <year>2020</year>
     <month>9</month>
     <day>14</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>dd/MM/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
   <property name="date">
    <date>
     <year>2021</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
